frontend/public/components/project-overview.jsx

```diff
@@ -0,0 +1,341 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as classNames from 'classnames';
+import { Button, EmptyState, Icon, Toolbar } from 'patternfly-react';
+import { Helmet } from 'react-helmet';
```

Comment:
Newline between `node_modules` imports and project local imports for consistency.